iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -66,11 +85,21 @@ Viewer.prototype._preProcessiXBRL = function(n, inHidden) {
         node = $(n).parent();
     }
     node.addClass("ixbrl-element").data('ivid',n.getAttribute("id"));
+    if (n.getAttribute("continuedAt")) {
+        node.data('iv-continued-at', n.getAttribute("continuedAt"));
```

Comment:
How is this `iv-continued-at` used?

Reply:
It's not.  A relic from a previous approach.  Removed in 9aef18b.